include/libuzfs_impl.h

```diff
@@ -96,9 +96,17 @@ struct libuzfs_zap_iterator {
 	zap_attribute_t za;
 };
 
-#define	UZFS_MAX_BLOCKSIZE	(1<<18)
-#define	UZFS_BONUS_LEN_DEFAULT	DN_BONUS_SIZE(512)
-#define	UZFS_BONUS_LEN_4K	DN_BONUS_SIZE(4096)
+#define	UZFS_MAX_BLOCKSIZE		(1<<18)
+
+// 320
+#define	UZFS_BONUS_LEN_DEFAULT		DN_BONUS_SIZE(512)
+#define	UZFS_MAX_RESERVED_DEFAULT	32
+#define UZFS_KV_CAPACITY_DEFAULT	256
+
+// 832
+#define	UZFS_BONUS_LEN_1K		DN_BONUS_SIZE(1024)
+#define UZFS_MAX_RESERVED_1K		200
+#define UZFS_KV_CAPACITY_1K		600
```

Comment:
these setting should be changed as offline discussion, also, let's add comments to describe the layout of dnode.